MiniDumper/CommandLineOptions.cs

```diff
@@ -28,6 +28,9 @@ class CommandLineOptions
         [Option('l', HelpText = "Display the debug logging of the process + diagnostics info from the minidumper.")]
         public bool Verbose { get; set; }
 
+        [Option('p', HelpText = "Memory commit threshold in MB at which to create a dump.")]
+        public int? MemoryCommitThreshold { get; set; }
```

Comment:
Could we rename it to `MemoryCommitThresholdInMB` and change type to `uint?`? Everyone will then know what is the unit.